.buildconfig.yml

```diff
@@ -80,6 +80,10 @@ projects:
       path: components/feature/findinpage
       description: 'Feature that will subscribe to the selected session and show an UI for results of find in page.'
       publish: true
+  feature-sitepermissions:
+    path: components/feature/sitepermissions
+    description: 'A feature that will subscribe to the selected session, and will provide an UI for all the incoming appPermission and contentPermission request.'
+    publish: true
```

Comment:
I think the description is more describing the implementation than what it actually does. :)

How about something like: "A feature for showing site permission request prompts".

Reply:
I like it :)